### PR TITLE
Binance: Update rate limits

### PR DESCRIPTION
**Pull request by ghost**
# PR Description

Came across some rate limit updates on binance when testing. Have just made some basic changes and modifications to rate limits in Binance. https://binance-docs.github.io/apidocs/spot/en/#change-log

Fixes # (issue)
Fixes getting banned from Binance since some common endpoints like getExchangeInfo and getOrders had their rate limits updated.

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [x] go test ./... -race
- [x] golangci-lint run
- [x] Created a test to send 100 requests in a very short amount of time for getExchangeInfo with previous rate limits and kept getting banned. Used the same test with new rate limits to check if it works.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/binance/ratelimit.go [hunk 1]
```diff
@@ -110,17 +113,21 @@ func (r *RateLimit) Limit(f request.EndpointLimit) error {
 		spotSymbolPriceAllRate:
 		limiter, tokens = r.SpotRate, 2
 	case spotHistoricalTradesRate,
-		spotAccountInformationRate,
 		spotOrderbookDepth500Rate:
 		limiter, tokens = r.SpotRate, 5
-	case spotOrderbookDepth1000Rate:
+	case spotOrderbookDepth1000Rate,
+		spotAccountInformationRate,
+		spotAllOrdersRate,
```
Comment (gloriousCode) [inline exchanges/binance/ratelimit.go:120]: `spotAllOrdersRate` needs to be part of the `SpotOrdersRate` rate limit group

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thank you MadCozBadd! Just two things I noticed

File: exchanges/binance/binance.go [hunk 2]
```diff
@@ -597,10 +597,9 @@ func (b *Binance) AllOrders(symbol currency.Pair, orderID, limit string) ([]Quer
 	if limit != "" {
 		params.Set("limit", limit)
 	}
-	if err := b.SendAuthHTTPRequest(exchange.RestSpotSupplementary, http.MethodGet, allOrders, params, spotOrdersAllRate, &resp); err != nil {
+	if err := b.SendAuthHTTPRequest(exchange.RestSpotSupplementary, http.MethodGet, allOrders, params, spotAllOrdersRate, &resp); err != nil {
```
Comment (gloriousCode) [inline exchanges/binance/binance.go:600]: Adding a `spotAllOrdersRate` has made `spotOrdersAllRate` unused. Please remove `spotOrdersAllRate`

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thank you for making those changes! One more thing 🎉

**Review by shazbert — COMMENTED** [review]
Nothing from me. Just waiting on glorious nits to be resolved.

**Comment by gloriousCode** [comment]
Thanks for making that change! The numbers are all correct, but I'm still getting banned when making open orders requests. Another PR may be required to sort out whats going on. Its hard to test when I've been banned 😄 I'll retest again another time.

**Review by gloriousCode — APPROVED** [review]
From further testing I can see that I can get banned on Binance when I have:
- a large amount of pairs enabled
- disabled the websocket 
- spamming getopenorders via CLI. 
 
I was not able to be banned when spamming getopenorders with a low amount of pairs and I could see the rate limiter appropriately adjusting request timeouts.

I don't think this PR should be blocked for such a scenario but it may be something we need to revisit when we look to trade across 1024 pairs simultaneously 😄 👀  _Not_ merging this PR will make it more likely for us to get banned anyway

tACK! Thank you madcozbadd!

**Review by shazbert — APPROVED** [review]
tACK!

**Review by thrasher- — APPROVED** [review]

**Pull request merged**